{
	"vulinfoid": "JVNDB-2016-004820",
	"vulinfodata": {
		"title": "Pivotal Cloud Foundry Ops Manager における脆弱性",
		"vulinfodescription": {
			"overview": "Pivotal Cloud Foundry (PCF) Ops Manager は、vCloud または vSphere が使用される場合、オペレータ用の SSH アクセスを適切に有効にしないため、不特定の影響およびリモート攻撃を受ける脆弱性が存在します。"
		},
		"affected": {
			"affecteditem": [
				{
					"name": "Greenplum",
					"productname": "Pivotal Cloud Foundry Operations Manager",
					"cpe": {
						"text": "cpe:/a:greenplum:cloud_foundry_ops_manager",
						"version": "2.2"
					},
					"versionnumber": [
						"1.6.17 未満",
						"1.7.8 未満の 1.7.x"
					]
				}
			]
		},
		"impact": {
			"cvss": [
				{
					"version": "2.0",
					"severity": {
						"text": "High",
						"type": "Base"
					},
					"base": "7.5",
					"vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P"
				},
				{
					"version": "3.0",
					"severity": {
						"text": "Critical",
						"type": "Base"
					},
					"base": "9.8",
					"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
				}
			],
			"impactitem": {
				"description": "不特定の影響およびリモート攻撃を受ける可能性があります。"
			}
		},
		"solution": {
			"solutionitem": {
				"description": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。"
			}
		},
		"related": {
			"relateditem": [
				{
					"type": "vendor",
					"name": "Pivotal Software, Inc.",
					"vulinfoid": "Pivotal Product Vulnerability Reports (CVE-2016-0897)",
					"url": "https://pivotal.io/security"
				},
				{
					"type": "vendor",
					"name": "Pivotal Software, Inc.",
					"vulinfoid": "CVE-2016-0897 Ops Manager vSphere and vCloud vulnerability",
					"url": "https://pivotal.io/security/cve-2016-0897"
				},
				{
					"type": "advisory",
					"name": "Common Vulnerabilities and Exposures (CVE)",
					"vulinfoid": "CVE-2016-0897",
					"url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-0897"
				},
				{
					"type": "advisory",
					"name": "National Vulnerability Database (NVD)",
					"vulinfoid": "CVE-2016-0897",
					"url": "http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2016-0897"
				},
				{
					"type": "cwe",
					"name": "JVNDB",
					"vulinfoid": "CWE-310",
					"url": "https://jvndb.jvn.jp/ja/cwe/CWE-310.html",
					"title": "暗号の問題"
				}
			]
		},
		"history": {
			"historyitem": [
				{
					"historyno": "0",
					"datetime": "2018-02-17T10:37:52+09:00",
					"description": "[2016年09月26日]\\n  掲載"
				}
			]
		},
		"datefirstpublished": "2016-09-26T15:54:28+09:00",
		"datelastupdated": "2016-09-26T15:54:28+09:00",
		"datepublic": "2016-06-24T00:00:00+09:00"
	}
}
